### PR TITLE
docs: Update join.md

### PR DESCRIPTION
**Pull request by richb-hanover**
Add a little more detail for the `join` parameters

### CODE REVIEW

File: web/book/src/reference/stdlib/transforms/join.md [hunk 1]
```diff
@@ -9,8 +9,10 @@ join side:{inner|left|right|full} table (condition)
 ## Parameters
 
 - `side` decides which rows to include, defaulting to `inner`.
-- Relation reference
-- A condition
+- `table` - a reference to a relation, possibly including an assignment, e.g.
+  `var= ...`
+- `condition` - a boolean condition
```
Comment (max-sixty) [inline web/book/src/reference/stdlib/transforms/join.md]:
```suggestion
- a reference to a relation, possibly including an assignment, e.g.
  `var= ...`
- a boolean condition
```

Only `side` is a named param, so removing the others as quoted names — is that reasonable?

**Review by max-sixty — APPROVED** [review]
Thanks!

File: web/book/src/reference/stdlib/transforms/join.md (hunk 1 above)
Comment (richb-hanover) [reply web/book/src/reference/stdlib/transforms/join.md]: Ahah! I was aiming for visual consistency to call out/mention each of the parameters. Try this...

Comment (max-sixty) [reply web/book/src/reference/stdlib/transforms/join.md]: But surely we do want to visually discriminate between `side` and the others?

File: web/book/src/reference/stdlib/transforms/join.md [hunk 2]
```diff
@@ -8,9 +8,11 @@ join side:{inner|left|right|full} table (condition)
 
 ## Parameters
 
-- `side` decides which rows to include, defaulting to `inner`.
-- Relation reference
-- A condition
+- _side_ decides which rows to include, defaulting to `inner`.
```
Comment (max-sixty) [inline web/book/src/reference/stdlib/transforms/join.md]:
```suggestion
- `side` specifies which rows to include, defaulting to `inner`.
```

How about this?

**Comment by max-sixty** [comment]
Thanks @richb-hanover !

**Pull request merged**